Gruntfile.js

```diff
@@ -73,21 +73,27 @@ module.exports = function(grunt) {
         const done = this.async();
         liveServer.start(serverParams).on('listening', done);
     });
-    
+
     grunt.registerTask('start-repl', function() {
         const finRepl = require(`./out/repl/index.js`);
         const done = this.async();
         finRepl.startRepl();
     });
-    
+
+    grunt.registerTask('doc', function() {
+        let exec = require('child_process').exec;
```

Comment:
This should go to the top of the page and be `const exec = require('child_process').exec;`